### ISSUE TITLE
508-defect-2 [HEADINGS] Headings are nested improperly in search results

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#24778]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**VFS Point of Contact:** _First name only: Angela

## User Story or Problem Statement:
As a user, I expect headings to be nested logically, much as they would be in a print report.

<img width="1792" alt="Screen Shot 2021-05-17 at 10 41 25 AM" src="https://user-images.githubusercontent.com/14154792/118507662-7112bf80-b6fc-11eb-9ce2-ca82fd37e071.png">

## Details
On the search results page, "Search va.gov," which precedes the search results, is a level two heading. All search results are level five headings with no level three or four headings in between.

note: this isn't specifically related to typeahead, but should be forwarded to the relevant team to get addressed

## Acceptance Criteria
- [x] All headings are nested sequentially.

**Comment by srsuddath** [comment]
This issue should be resolved as part of the search UI consistency ticket, and part of this PR:
https://github.com/department-of-veterans-affairs/vets-website/pull/17586
